packages/valory/skills/market_creation_manager_abci/rounds.py

```diff
@@ -689,6 +690,42 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Enum]]:
         return None
 
 
+class CloseMarketsRound(CollectSameUntilThresholdRound):
+    """CloseMarketsRound"""
+
+    payload_class = CloseMarketsPayload
+    synchronized_data_class = SynchronizedData
+    done_event = Event.DONE
+    no_majority_event = Event.NO_MAJORITY
+    collection_key = "content"
+
+    NO_TX = "no_tx"
+    ERROR_PAYLOAD = "error"
+
+    def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Enum]]:
```

Comment:
Is it possible to call super() here instead of re-write the method?

Reply:
i dont think so, we have custom logic

Reply:
Please @Adamantios take a look.

Reply:
It's possible to convert it to something like the following:

https://github.com/valory-xyz/trader/blob/7f158de664179d5d6e21a9e0d17f6ae0b8d1879c/packages/valory/skills/decision_maker_abci/states/redeem.py#L54-L58

Reply:
It would just mean moving the logic from here to the base class. Which i don't see what the point is in doing that right now, we are not reusing this.